src/main/java/uk/gov/hmcts/reform/bulkscan/orchestrator/services/ccd/AttachCaseCallbackService.java

```diff
@@ -53,13 +59,26 @@
 
     private static final Logger log = LoggerFactory.getLogger(AttachCaseCallbackService.class);
 
+    private final ServiceConfigProvider serviceConfigProvider;
+
     private final CcdApi ccdApi;
 
     private final PaymentsProcessor paymentsProcessor;
 
-    public AttachCaseCallbackService(CcdApi ccdApi, PaymentsProcessor paymentsProcessor) {
+    private final CreateCaseValidator createCaseValidator;
+
+    private final CcdCaseUpdater ccdCaseUpdater;
+
+    public AttachCaseCallbackService(ServiceConfigProvider serviceConfigProvider, CcdApi ccdApi,
+                                     PaymentsProcessor paymentsProcessor,
+                                     CreateCaseValidator createCaseValidator,
```

Comment:
I don't think we should have a `CreateCaseValidator` here - it's for case creation.